typography.html

```diff
@@ -18,10 +18,6 @@
 
     /* Shared Styles */
 
-    /*
```

Comment:
Or leave a note around the commented out `@applies` as to why they're commented out? Basically, I don't want this comment to be lost, if @morethanreal left it in here for a reason.